src/sageworks/web_components/experiments/assets/dashAgGridComponentFunctions.js

```diff
@@ -0,0 +1,9 @@
+var dagcomponentfuncs = (window.dashAgGridComponentFunctions = window.dashAgGridComponentFunctions || {});
+
+dagcomponentfuncs.Link = function (props) {
+    return React.createElement(
+        'a',
+        {href:  props.value},
+        props.value
+    );
+};
```

Comment:
This is good, I'm a little surprised that we need to create JavaScript just to have an HTML link in AgGrid.

Reply:
Yes that the easiest custom cell component possible, so that seems a lot of work but it's really powerful, you can put really anything you have in mind in the cells, take a look https://dash.plotly.com/dash-ag-grid/cell-renderer-components. You could put even a dag table in cells 😆

Anyway, if you prefer it should be possible to use directly markdown in dag, setting `dangerously_allow_code=True` for the table.